### ISSUE TITLE
seg fault on OSX 10.14.6 (Mojave) with ROOT 6.24.04 installed through homebrew

### ISSUE THREAD

**Issue by villaa** [villano-lab/nrCascadeSim#32]
I often (but not always!?) get strange errors when running with the setup (although everything compiles thanks to the PR #28). 

The errors look like:

```
(base) Anthonys-MacBook-Pro:bin villaa$ ./realizeCascades -n 10000 -o output.root ../levelfiles/allge_ngam_WFast.txt 
Seed used: 16807
MTRand: 0x7f803b831c00
**************../levelfiles/allge_ngam_WFast.txt***************

Cascade ID: 1/74
Fraction of this cascade: 0
Neutron separation: 6.92625e-310
Mass number: 4
Number of steps: 0

Energy Levels (keV)	|	tau (fs)
------------------------------------------------
      *****       	 	 ***** 

Cascade ID: 2/74
Fraction of this cascade: 4.743e-322
Neutron separation: 2.1565e-314
Mass number: 96
Number of steps: 0

Energy Levels (keV)	|	tau (fs)
------------------------------------------------
      *****       	 	 ***** 

Cascade ID: 3/74
Fraction of this cascade: 0
Neutron separation: 2.1565e-314
Mass number: 488
Number of steps: 7632997

Energy Levels (keV)	|	tau (fs)
------------------------------------------------
      *****       	 	 ***** 

 *** Break *** segmentation violation
      -5.1932e+214       	 	 [/usr/lib/system/libsystem_platform.dylib] _sigtramp (no debug info)
[<unknown binary>] (no debug info)
[/usr/lib/system/libdyld.dylib] start (no debug info)
```

It's clear that something is going wrong with reading the file because of the whack values for the neutron separation energy in the first record.

**Comment by villaa** [comment]
BTW the version I used to link to was 6.22.08 the direct URL is in [this link](https://raw.githubusercontent.com/Homebrew/homebrew-core/31f9ba288739a51432852af030a8f92dd8b21794/Formula/root.rb).

**Comment by villaa** [comment]
Note: found out that ROOT 6.16/00 uses C++14

**Comment by villaa** [comment]
Addressed by PR #61 -- we decided to support ROOT versions 6.20.00 through 6.24.06.
